compiler/rustc_hir_typeck/src/fn_ctxt/checks.rs

```diff
@@ -2484,7 +2485,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                         }
                         generics_with_unmatched_params.push(generic_param);
                     } else {
-                        spans.push_span_label(param.span, "");
+                        spans.push_span_label(param.span(), "");
```

Comment:
This is the line adding all the extra spans to the diagnostic. It looks like it's just a catch-all for parameters that aren't otherwise matched.

Given that the final commit already affects a pile of tests should I just go ahead and remove it?